src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -720,6 +720,26 @@ else if (getDriver().getTitle().startsWith("Sign In"))
                     // Note: leave the self-report setting unchanged
                     customizeSitePage.save();
                 }
+
+                //Waiting for search service to boot up
+                Connection connection = createDefaultConnection();
+
+                SimpleGetCommand command = new SimpleGetCommand("search", "json");
+                command.setParameters(Map.of("q", "pinging to check server is started", "scope", "All"));
+                CommandResponse searchResponse;
+                try
+                {
+                    do
```

Comment:
Consider adding a polling interval to avoid spamming the service- 
Also consider building an exit condition to the loop- if the service never responds 200, this code should probably have a built-in exit after some number of iterations or time (or both)

Reply:
Good point